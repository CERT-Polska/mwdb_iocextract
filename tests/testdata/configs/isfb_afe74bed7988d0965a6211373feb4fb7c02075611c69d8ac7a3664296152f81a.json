{"compilation_date": "Mar 17 2020", "public_key": {"n": "11752370188818234234367932984885459542854166340237087662006434322123117188694577648403840181786403017994618634701561673771068638661466408611383826900169189", "e": 65537}, "domains": [{"cnc": "settings-win.data.microsoft.com"}, {"cnc": "accrualdewd.xyz"}], "botnet": 5020, "server": 12, "key": "32291667LSJUYNUU", "timer": 0, "dga_base_url": "constitution.org/usdeclar.txt", "dga_crc": 1320669898, "dga_tld": [".com", ".ru", ".org"], "dga_season": 10, "exe_type": "loader", "dga_count": 5, "dga_lsa_seed": 3988359472, "dga_seed": 1, "version": "2.14.131", "xcookie": -1, "type": "isfb"}