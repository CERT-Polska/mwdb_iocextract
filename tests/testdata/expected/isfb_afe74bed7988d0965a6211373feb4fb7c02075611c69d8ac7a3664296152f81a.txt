RsaKey n=11752370188818234234367932984885459542854166340237087662006434322123117188694577648403840181786403017994618634701561673771068638661466408611383826900169189 e=65537
NetLoc settings-win.data.microsoft.com
NetLoc accrualdewd.xyz
